ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -29,12 +29,13 @@
 
 .. code-block:: python
 
-    from opentelemetry.ext.flask import FlaskInstrumentor
-    FlaskInstrumentor().instrument()  # This needs to be executed before importing Flask
     from flask import Flask
+    from opentelemetry.ext.flask import FlaskInstrumentor
 
     app = Flask(__name__)
 
+    FlaskInstrumentor().instrument(app=app)
```

Comment:
```suggestion
    FlaskInstrumentor().instrument_app(app)
```

?

Reply:
Weird, I am sure I changed this... fixing.